src/readers/stream.ts

```diff
@@ -37,6 +49,10 @@ export default class ReaderStream extends Reader<NodeJS.ReadableStream> {
 		};
 
 		for (let i = 0; i < filepaths.length; i++) {
+			if (stream.writableEnded) {
```

Comment:
The `writableEnded` property exist in the Stream only start with Node.js 12.9.0+. The `fast-glob` package supports Node.js 8+. This works now because `writableEnded` has `undefined` value.